Sources/Login/LoginCoordinator.swift

```diff
@@ -81,6 +83,7 @@ final class LoginCoordinator: NSObject,
                 } else {
                     let networkErrorScreen = errorPresenter
                         .createNetworkConnectionError(analyticsService: analyticsCenter.analyticsService) { [unowned self] in
+                            introViewController?.enableIntroButton()
```

Comment:
Because we haven't launched the `AuthenticationCoordinator` at this point, we need to ensure this is called on the network error button action.